packages/client/src/components/Form.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+const Form = styled.form.attrs((props) => ({
+  padding: props.padding || "2rem",
+  // 18% 4rem
+}))`
```

Comment:
I didn't know styled-components could do this cool!

Reply:
yeah! it was interesting to see how much flexibility it brings to components.